jumpcutter.py

```diff
@@ -1,3 +1,4 @@
+#!/bin/env python3
```

Comment:
Is this line truly needed for running on linux? Because for my windows installation this screws me over, as now the python launcher py.exe tries to look in the specified directory (which does not exist) and thus fail to run.

Reply:
So I looked over it, and apparently the launcher has some default behavior for *some* of the shabang lines.
See https://docs.python.org/3/using/windows.html#shebang-lines. While it also mentions that one could change the settings of the py launcher, I want this script to work out of the box (on windows) without weird edge cases on the launching behaviour.

tldr: For `#!/usr/bin/env` the py.exe launcher actually also looks through the windows PATH env variable whether it finds a suitable python installation. However since by default those are only named `python.exe` and not `python3.exe` we would hence need to change the line to:
```suggestion
#!/usr/bin/env python
```

While this removes the constraint of the file needing python 3, I believe this should be a good compromise over outright removing the line :)